src/pages/StyleGuide.js

```diff
@@ -20,6 +22,21 @@ class StyleGuide extends Component {
   render() {
     return div({ style: { paddingLeft: '1rem', paddingRight: '1rem' } }, [
       h1('Style guide'),
+      div({ style: styles.container }, [
+        div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+          _.map(Style.colors, (v, k) =>
+            div({
+              style: {
+                backgroundColor: v,
+                width: 150, height: 50,
+                display: 'flex', justifyContent: 'center', alignItems: 'center'
+              }
+            }, [
+              span({ style: { backgroundColor: 'white', padding: 2 } }, k)
```

Comment:
Since this is already a flex, why not just use `div`s here? It would remove the import line change

Reply:
No good reason, just details of how the code evolved. Will fix